Misc-Tools.lua

```diff
@@ -117,66 +161,102 @@ local function OnCreateMove(pCmd)
 
     -- Random Fakelag
     if mRandLag:GetValue() == true then
-        if pLocal:IsAlive() and pCmd.command_number % 20 == 0 then
-            local randValue = math.random(120, 315)
+        flTimer = flTimer +1
+        if (flTimer >= mRandLagValue:GetValue()) then
```

Comment:
Why not use `pCmd.command_number % mRandLagValue:GetValue() == 0`?

Reply:
Hmm you're right. I just don't like having multiple variables only used for counting, but it's better than using command_number i guess